test/microbench/org/apache/cassandra/test/microbench/ZeroCopyStreamingBenchmark.java

```diff
@@ -145,7 +145,6 @@ public void setupBenchmark() throws IOException
                                      .withFirstKey(sstable.first)
                                      .withTableId(sstable.metadata().id)
                                      .build();
-
```

Comment:
@isaacreath I know it is super annoying but if you manage to put these lines back (on line 171 too), this file would stayed unchanged and your patch would be smaller. I totally get that this seems like a lot of red tape but we need to contain the dragon / lower the entropy as much as we can.

Reply:
cc @pauloricardomg